apps/backend/app/scrap/scrap.py

```
@@ -67,8 +67,31 @@ def main():
+                    dong = p
+            row["gu"]   = gu
+            row["dong"] = dong
+        # ────────────────────────────────────────────────────────────────
+
+
+        data = [row for row in data if row["school_name"] != "평균"]
+        def _avg(row):
+            v = row.get("average", "").replace("%", "").strip()  # '97.6%' → '97.6'
+            return float(v) if v else -1                         # 빈 값이면 -1
+        data.sort(key=_avg, reverse=True)
+        for r in data: r.pop("average", None)
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)